### ISSUE TITLE
Change Request: Support <style>-tag linting, eg. in Svelte files

### ISSUE THREAD

**Issue by uh-zuh** [eslint/css#65]
### Environment

Svelte

### What problem do you want to solve?

For example Svelte-files have structure like this:
```svelte
<script lang="ts">
    console.log(123);
</script>

<div class="red">
    234
</div>

<style>
    .red {
        color: red;
    }
</style>

```

It is good to support linting CSS in `<style>`-tag in such files.

### What do you think is the correct solution?

Linting `<style>`-tag in Svelte-files.

### Participation

- [ ] I am willing to submit a pull request for this change.

**Comment by nzakas** [comment]
Thanks for the suggestion. This is something that needs to be implemented as part of ESLint, not as part of this plugin. You can follow the RFC here:
https://github.com/eslint/rfcs/pull/105
